tests/integration/test_mysql_protocol/java_client_test.sql

```diff
@@ -17,7 +17,7 @@ CREATE OR REPLACE TABLE ps_simple_data_types
     b     Boolean
 ) ENGINE MergeTree ORDER BY i8;
 
-INSERT INTO ps_simple_data_types
+INSERT INTO simple_data_types
```

Comment:
Removed the `ps_` (prepared statements) prefix as this dataset is now used for both text/binary protocols.